pkg/docker/config/config_test.go

```diff
@@ -675,3 +676,170 @@ func TestAuthKeysForRef(t *testing.T) {
 		require.Equal(t, tc.expected, result, tc.name)
 	}
 }
+
+func TestSetCredentials(t *testing.T) {
+	for _, tc := range []struct {
+		registries []string
+		assert     func(dockerConfigFile)
```

Comment:
Non-blocking: This looks like it could be just an `expectedKeys []string`, or something like that.

Reply:
Yeah, not any more after my modification of the `assert` function. :upside_down_face: